### PR TITLE
V0.1.0/event pagination

### PR DESCRIPTION
**Pull request by duypv98**
Check new API spec (get events by uid)

### CODE REVIEW

File: db/odm/event.odm.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import Group from 'db/models/Group';
 async function findByUserId(userId) {
   const listEvents = await Group.find(
     { users: userId },
+    { _id: 0 },
```
Comment (namdaoduy) [inline db/odm/event.odm.js]: Why adding `{ _id: 0 }` here?

**Review by namdaoduy — CHANGES_REQUESTED** [review]

File: db/odm/event.odm.js (hunk 1 above)
Comment (duypv98) [reply db/odm/event.odm.js]: As it is Group _id, that is unnecessary

File: common/error.js [hunk 2]
```diff
@@ -74,3 +64,22 @@ export class TokenExpiredError extends ServerAPIError {
...
+
+export class UserNotFoundError extends ServerAPIError {
+  constructor({ ...payload }) {
+    super({
+      code: 40401,
+      message: 'User Not Found',
+      ...payload,
+    });
+  }
+}
```
[the remark this answers is not in the document]
Comment (duypv98) [reply common/error.js]: solved https://github.com/EasyEventV2/ee2_api/pull/5/commits/dcc634a1ccb0210f3a575316c1933531e0dfa246

[the remark this answers is not in the document]
Comment (duypv98) [reply common/error.js]: https://github.com/EasyEventV2/ee2_api/pull/5/commits/69f0baf704721b8cf494f97228d6737f8742fd87

File: db/odm/event.odm.js (hunk 1 above)
Comment (duypv98) [reply db/odm/event.odm.js]: https://github.com/EasyEventV2/ee2_api/pull/5/commits/69f0baf704721b8cf494f97228d6737f8742fd87

**Review by namdaoduy — APPROVED** [review]

**Pull request merged**